src/commands/synthetics/utils/internal.ts

```diff
@@ -140,6 +142,26 @@ type AccumulatorBaseConfigOverride = Omit<
   cookies?: Partial<Exclude<UserConfigOverride['cookies'], string>>
 }
 type AccumulatorBaseConfigOverrideKey = keyof AccumulatorBaseConfigOverride
+const allOverrideKeys = [
```

Comment:
Let's make so that this list is required to be updated when we add an override key.
```suggestion
const allOverrideKeys: AccumulatorBaseConfigOverrideKey[] = [
```

I tried to make it throw a compilation type error when we remove an override key as well, but it's getting too complex for this.